flink-runtime/src/test/java/org/apache/flink/runtime/leaderelection/ZooKeeperLeaderElectionTest.java

```diff
@@ -70,7 +68,8 @@
 import static org.mockito.Mockito.when;
 
 /**
- * Tests for the {@link ZooKeeperLeaderElectionService} and the {@link ZooKeeperLeaderRetrievalService}.
+ * Tests for the {@link DefaultLeaderElectionService} with {@link ZooKeeperLeaderElectionDriver}
+ * and the {@link DefaultLeaderRetrievalService} with {@link org.apache.flink.runtime.leaderretrieval.ZookeeperLeaderRetrievalDriver}.
  */
 public class ZooKeeperLeaderElectionTest extends TestLogger {
```

Comment:
It might be possible to update some of the tests to work directly against the `ZooKeeperDrivers` instead of having to instantiate a the services.